docs/Other SDL Features/Setting the Built-In Navigation Destination/index.md

```diff
@@ -0,0 +1,155 @@
+# Setting the Built-In Navigation Destination
+The `SendLocation` RPC gives you the ability to send a GPS location to the embedded navigation app on the head unit. When the request is sent the user will be prompted to navigate to that location using the embedded navigation app. 
```

Comment:
Remove all references to the "embedded" navigation and instead reference the "active" navigation.